includes/Modules/Idea_Hub.php

```diff
@@ -254,10 +254,11 @@ private function admin_notice_idea_hub_ideas( $notices ) {
 			array(
 				'content'         => function() {
 					return sprintf(
-						'<p>%s <a href="%s">%s</a></p>',
-						esc_html__( 'Need some inspiration? Revisit your saved ideas in Site Kit.', 'google-site-kit' ),
+						'<p>%s <a href="%s">%s</a> %s</p>',
+						esc_html__( 'Want some inspiration for a new post?', 'google-site-kit' ),
 						esc_url( $this->context->admin_url() . '#saved-ideas' ),
-						esc_html__( 'See saved ideas', 'google-site-kit' )
+						esc_html__( 'Revisit your saved ideas', 'google-site-kit' ),
+						esc_html__( 'in Site Kit.', 'google-site-kit' )
```

Comment:
@asvinb it should be one translatable string:

```php
$message = sprintf(
    /* translators: %s: URL to saved ideas */
    __( 'Want some inspiration for a new post? <a href="%s">Revisit your saved ideas</a> in Site Kit.', 'google-site-kit' ),
    esc_url( $this->context->admin_url() . '#saved-ideas' ),
);

return "<p>{$message}</p>"
```

Same should be done for the new ideas notification.

Reply:
👍 Ready for another pass @eugene-manuilov